### DISCUSSION TITLE
Translation Fix. Best way to pull request

### DISCUSSION THREAD

**Discussion by gabriel-quina** [chatwoot/chatwoot#8289] [Translations]
Hello everyone, everything good?

I'm correcting some inconsistencies in the translation into Portuguese and Brazilian Portuguese and I would like to know the best way to make the pull request. Should I create a new branch or commit directly to the develop branch?

**Comment by pranavrajs** [comment]
https://www.chatwoot.com/docs/contributing-guide/translation-guidelines
